docs/modules/REST.md

```diff
@@ -89,6 +89,19 @@ $I->amNTLMAuthenticated('jon_snow', 'targaryen');
  * `[Part]` xml
 
 
+### amAWSAuthenticated
+Allows to send REST request using AWS Authorization
+```php
+<?php
+$I->amAWSAuthenticated('awsKey', 'awsSecret', 'awsService', 'awsRegion');
+?>
+```
+ * `param` $key
+ * `param` $secret
+ * `param` $service
+ * `param` $region
+
+
```

Comment:
This documentation file is autogenerated from REST.php, so this change will be lost.
Please move your code example to REST.php

Reply:
Removed from Docs